model/db_schema.sql

```diff
@@ -49,6 +49,35 @@ CREATE TABLE Year (
     FOREIGN KEY (user_id) REFERENCES Tutor (user_id) ON UPDATE CASCADE
 );
 
+/*
+Test Data
+@author Oleg
+*/
+
+INSERT INTO User Values (default,"Bobby@mail.com","bobTroll!!",0),(default,"Andy@mail.com","AndyHot!!",0),(default,"CoolGuy@mail.com","notCool!!",0),
```

Comment:
im worried your test data is going to conflict with Dallas's test data. If you haven't already, please test on your workbench with both sets of data together to make sure it doesn't throw errors.

Reply:
works like a charm now

Reply:
Thanks for adding to the test data